### ISSUE TITLE
What is the exact setting for Panoptic FPN pretrained weight?

### ISSUE THREAD

**Issue by AumXIV** [facebookresearch/detectron2#2309]
## ❓ How to do experiment as pretrained weight?

I'm trying to reproduce an experiment exactly the same as pretrained weight but I have an environment shown below:

```
----------------------  ------------------------------------------------------------------------------
sys.platform            linux
Python                  3.8.3 | packaged by conda-forge | (default, Jun  1 2020, 17:43:00) [GCC 7.5.0]
numpy                   1.19.2
detectron2              0.3 @/home/wen/anaconda3/envs/mrcnn/lib/python3.8/site-packages/detectron2
Compiler                GCC 7.3
CUDA compiler           CUDA 9.2
detectron2 arch flags   3.7, 5.0, 5.2, 6.0, 6.1, 7.0
DETECTRON2_ENV_MODULE   <not set>
PyTorch                 1.7.0 @/home/wen/anaconda3/envs/mrcnn/lib/python3.8/site-packages/torch
PyTorch debug build     True
GPU available           True
GPU 0                   GeForce GTX 1080 (arch=6.1)
CUDA_HOME               /usr/local/cuda
Pillow                  8.0.1
torchvision             0.8.1 @/home/wen/anaconda3/envs/mrcnn/lib/python3.8/site-packages/torchvision
torchvision arch flags  3.5, 5.0, 6.0, 7.0
fvcore                  0.1.2.post20201111
cv2                     4.5.0
----------------------  ------------------------------------------------------------------------------
PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.2 Product Build 20200624 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v1.6.0 (Git Hash 5ef631a030a6f73131c77892041042805a06064f)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 9.2
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_37,code=compute_37
  - CuDNN 7.6.3
  - Magma 2.5.2
  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_VULKAN_WRAPPER -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, 
```

Since I have one GPU the config will change:
```
SOLVER:
  AMP:
    ENABLED: False
  BASE_LR: 0.0025
  BIAS_LR_FACTOR: 1.0
  CHECKPOINT_PERIOD: 5000
  CLIP_GRADIENTS:
    CLIP_TYPE: value
    CLIP_VALUE: 1.0
    ENABLED: False
    NORM_TYPE: 2.0
  GAMMA: 0.1
  IMS_PER_BATCH: 2
  LR_SCHEDULER_NAME: WarmupMultiStepLR
  MAX_ITER: 270000
  MOMENTUM: 0.9
  NESTEROV: False
  REFERENCE_WORLD_SIZE: 0
  STEPS: (210000, 250000)
  WARMUP_FACTOR: 0.001
  WARMUP_ITERS: 1000
  WARMUP_METHOD: linear
  WEIGHT_DECAY: 0.0001
  WEIGHT_DECAY_BIAS: 0.0001
  WEIGHT_DECAY_NORM: 0.0
```
I have tried the training until 124999 iterations and got the result below:
```
Training in 124999 iterations 
|        |   PQ   |   SQ   |   RQ   |  #categories  |
|:------:|:------:|:------:|:------:|:-------------:|
|  All   | 28.426 | 71.592 | 35.687 |      133      |
| Things | 35.211 | 77.538 | 43.402 |      80       |
| Stuff  | 18.185 | 62.619 | 24.042 |      53       |
```
And here is the pretrained weight
```
Pretrained weight
|        |   PQ   |   SQ   |   RQ   |  #categories  |
|:------:|:------:|:------:|:------:|:-------------:|
|  All   | 43.019 | 79.992 | 52.064 |      133      |
| Things | 49.696 | 82.879 | 59.220 |      80       |
| Stuff  | 32.940 | 75.633 | 41.263 |      53       |
```
As you can see the result have a big gap. 

The question is :
1. What is the exact setting for pertrained weight? Is it the same as the default config file?
2. In the paper has said about lambda i and lambda s which is the weight for both tasks loss, what is the parameter setting?
3. How many iterations and time for the experiment?

**Comment by ppwwyyxx** [comment]
> What is the exact setting for pertrained weight? Is it the same as the default config file?

answered in https://github.com/facebookresearch/detectron2/blob/master/MODEL_ZOO.md#how-to-read-the-tables

> In the paper has said about lambda i and lambda s which is the weight for both tasks loss, what is the parameter setting?

`MODEL.PANOPTIC_FPN.INSTANCE_LOSS_WEIGHT, MODEL.SEM_SEG_HEAD.LOSS_WEIGHT`.

> How many iterations and time for the experiment?
answered in https://github.com/facebookresearch/detectron2/blob/master/MODEL_ZOO.md#how-to-read-the-tables
